### DISCUSSION TITLE
How to improve GPU-Utilization

### DISCUSSION THREAD

**Discussion by TonJ24** [open-mmlab/mmediting#1190] [How-to] [answered]
I find the code based on mmediting has low utilization of GPU. 
Maybe it attributes to load dataset, eta.
I wanna to know how to enhance the rate of SM?

**Comment by wangruohui** [comment, marked as answer]
This is quite a general question because GPU utilization can be affected by many factors including data loading from the disk, data preprocessing (i.e. pipeline) which rely on CPU, and even the model it self. 

Specifically, you can check `data_time` in the log to see if some bottleneck at data loading exists and maybe the overall system loading to see if some other bottleneck exists.

**Comment by TonJ24** [comment]
> This is quite a general question because GPU utilization can be affected by many factors including data loading from the disk, data preprocessing (i.e. pipeline) which rely on CPU, and even the model it self.
> 
> Specifically, you can check `data_time` in the log to see if some bottleneck at data loading exists and maybe the overall system loading to see if some other bottleneck exists.

Thanks for your reply, I will try to check up it.
